DESCRIPTION

```diff
@@ -9,7 +9,8 @@ Encoding: UTF-8
 Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.3.2
 Depends: R (>= 4.0.0),
-         patchwork
+         patchwork,
+         ambient
```

Comment:
I am compelled to suggest putting "ambient" on top of "patchwork" :sweat_smile: 

```suggestion
         ambient,
         patchwork
```

Reply:
I don't understand why but I don't object.

Reply:
just because "a" comes before "p" haha